modules/govuk/manifests/apps/whitehall.pp

```diff
@@ -332,6 +337,9 @@
     # Protocol relative URL so assets in admin are on the same domain but work
     # in production and development. (This is needed for IE8)
     govuk::app::envvar {
+      "${title}-ASSET_MANAGER_BEARER_TOKEN":
+        varname => 'ASSET_MANAGER_BEARER_TOKEN',
+        value   => $asset_manager_bearer_token;
```

Comment:
Minor: It looks as if the comment beginning "Protocol relative URL..." belongs with the definition of `GOVUK_ASSET_ROOT`. By adding the definition of `ASSET_MANAGER_BEARER_TOKEN` here, I think we're making things worse by further separating the comment from its related code.

Reply:
Minor: Did you consider putting the `` definition in the other `govuk::app::envvar` block further down which includes the definition of `PUBLISHING_API_BEARER_TOKEN`? Is this definitely the right place?

Reply:
> It looks as if the comment beginning "Protocol relative URL..." belongs with the definition of `GOVUK_ASSET_ROOT`...

I've added a new commit to make it clear that the comment relates to `GOVUK_ASSET_ROOT`. I've left the `ASSET_MANAGER_BEARER_TOKEN` where it is as the list of environment variables in that block are in alphabetical order.